frontend/openchat-shared/src/utils/logging.ts

```diff
@@ -55,3 +55,7 @@ export function debug<T>(data: T, msg?: string): T {
     }
     return data;
 }
+
+export function logDuration(msg: string, started: number): void {
```

Comment:
We're not referencing this function in the PR. I suppose it doesn't really matter - it might be prove useful later.

Reply:
Yes I thought I would just leave it there since I have deleted it and recreated it a couple of times recently.